tests/write_font_test.py

```diff
@@ -329,15 +337,18 @@ def test_ufo_color_base_glyph_bounds(svgs, config_overrides, expected):
     config, glyph_inputs = test_helper.color_font_config(config_overrides, svgs)
     ufo, _ = write_font._generate_color_font(config, glyph_inputs)
 
-    base_glyph_name = "e000"
-    assert len(ufo[base_glyph_name]) == 0
+    base_glyph_names = [f"e{str(i).zfill(3)}" for i in range(len(svgs))]
```

Comment:
(apparently I can't comment unmodified lines)

I think maybe you should rename expected, it wasn't a very good name to begin with and now it's even less clear. expected_clip_boxes perhaps?

Reply:
Done https://github.com/googlefonts/nanoemoji/commit/a365e618a8595d261c6c6159b4384632582a463e